content/en/docs/v3.4/get-started/etcd.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Get Started with etcd
+weight: 910
+description: Getting started with etcd on Linux or macOS
+---
+
+# Quickstart
+
+This guide shows how to download, setup, and test a simple single-member cluster of etcd.
```

Comment:
There's no setup per se. I'd drop this text too. I think that the page description will be enough.